survey/2-learning-and-education.yml

```
@@ -58,4 +79,17 @@ questions:
       - Online Communities (StackOverflow/Reddit/Twitter)
       - AI Assistant (ChatGPT, etc)
       - Open source contributions
+      - Coding challenges (LeetCode, HackerRank, etc)
       - Other platforms
+
+  - label: What's your biggest challenge in learning new technologies?
```

Maybe we can phrase this to a multiple choice question ? With a limit of 3/4 choices (if possible) ?


```suggestion
  - label: What are the biggest challenges you find in learning new technologies? (pick 3)
```